src/stavenote.ts

```diff
@@ -486,7 +552,7 @@ export class StaveNote extends StemmableNote {
 
       // Calculate displacement of this note
       const line = props.line;
-      if (lastLine === null) {
+      if (!lastLine) {
```

Comment:
This is bad because `lastline` can be `0`. Please check the other false tests too for `number`.

Reply:
That is right, checked againt undefined now.